src/components/homePage/Footer.vue

```diff
@@ -22,9 +20,9 @@
 </template>
 
 <script>
-import Facebook from "../../assets/socials/facebook.svg";
-import Instagram from "../../assets/socials/instagram.svg";
-import Twitter from "../../assets/socials/twitter.svg";
+import Facebook from "@/assets/socials/facebook.svg";
+import Instagram from "@/assets/socials/instagram.svg";
+import Twitter from "@/assets/socials/twitter.svg";
```

Comment:
Is there a reason why you choose to use `@` instead of the dots. I feel like the latter would help someone understand where the icons are being exactly imported from

Reply:
@ in Vue projects refers to the root of the source files, in this case its `src`. I think @ is generally considered better because it means that if we were to move this file it's paths wouldn't need to be updated. (Also, I think VS code will recognize these paths and update them if we were to ever move the file)